### DISCUSSION TITLE
Cfclient does not open in windows 10

### DISCUSSION THREAD

**Discussion by Staphord** [bitcraze/discussions#694] [Q&A]
According to the documentation, I have installed Cfclient, but I am still unable to open it. it state 
  'cfclient' is not recognized as an internal or external command,
operable program or batch file.

I have python 3.10  installed. 
Kindly help.

![image](https://user-images.githubusercontent.com/56459243/235391212-00170ac1-3d4b-4a06-bd55-91aa7da000e9.png)
![image](https://user-images.githubusercontent.com/56459243/235391411-0cac2028-c9b8-496a-96dd-459a674642d2.png)

**Comment by ataffanel** [comment]
It seems that the location where python install executable for scrips is not in your path.

One other way to run the client is directly with the python command which you should have in path since pip is. You can run: `python -m cfclient.gui`.

**Reply by knmcguire** [reply]
@Staphord did this help your issue? If so, please mark it as an answer please
